src/ldc/eh/msvc.d

```diff
@@ -109,12 +111,16 @@ extern(C) void _d_throw_exception(Object e)
         if (!old_terminate_handler)
             old_terminate_handler = set_terminate(&msvc_eh_terminate);
     }
-    auto t = cast(Throwable) e;
-    exceptionStack.push(t);
+    
+    auto throwable = cast(Throwable) e;
+    exceptionStack.push(throwable);
+
+    if (throwable.info is null && cast(byte*)throwable !is typeid(throwable).init.ptr)
+        throwable.info = _d_traceContext();

```

Comment:
Even though this is the behaviour used on other platfroms or in dmd, I don't think it should be encouraged: it can make exceptions unusable slow, even if only used in rare cases. The trace should only be created with the catch statement that actually wants to print it. That's quite a bit more difficult to implement, though...